### DISCUSSION TITLE
non-ascii character bug in npm

### DISCUSSION THREAD

**Discussion by Notenlish** [community/community#49162] [Actions]
### Select Topic Area

Bug

### Body

```Microsoft Windows [Version 10.0.22621.1265]
(c) Microsoft Corporation. Tüm hakları saklıdır.

C:\Users\İhsan>C:\Users\İhsan\projects\testproject\dev
'C:\Users\İhsan\projects\testproject\dev' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\İhsan>cd C:\Users\İhsan\projects\testproject\dev

C:\Users\İhsan\projects\testproject\dev>npm install
npm ERR! Cannot read properties of null (reading 'edgesOut')

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\İhsan\AppData\Local\npm-cache\_logs\2023-03-04T11_38_14_162Z-debug-0.log
```
I'm trying to use the install command in npm but its not working. I've looked online and appearently the reason is caused by non-ascii characters in my path. The problem is that "İ" in my username is the character that's causing this trouble. 

I tried setting the unicode in npm config to true but it still didn't work.
```
PS C:\Users\İhsan> cd projects/testproject/dev
PS C:\Users\İhsan\projects\testproject\dev> npm install
npm ERR! Cannot read properties of null (reading 'edgesOut')

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\İhsan\AppData\Local\npm-cache\_logs\2023-03-04T11_45_32_082Z-debug-0.log
PS C:\Users\İhsan\projects\testproject\dev> npm config get unicode
true
```

The full error log is here:
[errorlog.txt](https://github.com/community/community/files/10888194/errorlog.txt)

Edit: I forgot to change the label, sorry

**Comment by jsoref** [comment]
This isn't the proper forum for problems with `npm`

In theory, `npm`'s cli lives here:
https://github.com/npm/cli

You might try posting to https://stackoverflow.com/ before you post to npm's tracker.

It certainly wouldn't shock me if this was a bug in `npm` itself, but if I were a nonnative speaker, I'd probably want to find someone who could help me a bit before I tried to file a bug straight to a major repository like that one.

**Reply by Notenlish** [reply]
Oh thanks I wasn't sure if this was the right place to post this I tried some stuff but still couldn't get it to work. I'll ask it on stackoverflow
